spec/models/manageiq/providers/azure_stack/cloud_manager/vcr_specs/refresher_spec.rb

```diff
@@ -143,6 +153,41 @@ def assert_specific_orchestration_stack
     )
   end
 
+  def assert_specific_network
+    expect(network).not_to be_nil
+    expect(ems_ref_suffix(network.ems_ref)).to match(%r{^/providers/microsoft.network/virtualnetworks/[^/]+$})
+
+    expect(network.cloud_subnets).not_to be_nil
+    expect(network.cloud_subnets.size).to eq(1)
+  end
+
+  def assert_specific_subnet
+    expect(subnet).not_to be_nil
+    expect(ems_ref_suffix(subnet.ems_ref)).to match(%r{^/providers/microsoft.network/virtualnetworks/[^/]+/subnets/[^/]+$})
+
+    expect(subnet.cloud_network).not_to be_nil
```

Comment:
This assertion is actually redundant because we assert the exact value in the next line anyway